src/lib/list/list.md

```diff
@@ -42,6 +42,20 @@ element in an `<mat-list-item>`.
 </mat-nav-list>
 ```
 
+### Action lists
+
+Use `mat-action-list` tags for action lists (i.e. lists that have button tags).
+
+Simple action lists can use the `mat-list-item` attribute on button tag elements directly:
+
+```html
+<mat-action-list>
+  <button mat-list-item  *ngFor="let link of links" (click)="alertItem(link.name)">
```

Comment:
1. Extra whitespace character after `mat-list-item`
2. Show what the `alertItem` function does

Reply:
removed the white space. Also removed the click part since it is not showing the key point here and may cause unnecessary confusion.